openbb_platform/core/openbb_core/provider/standard_models/spot.py

```diff
@@ -32,7 +32,6 @@ class SpotRateQueryParams(QueryParams):
     category: str = Field(
         default="spot_rate",
         description="Rate category. Options: spot_rate, par_yield.",
-        json_schema_extra={"choices": ["par_yield", "spot_rate"]},
```

Comment:
why removing the standard choices?

Reply:
Because these shouldn't be in the standard model.